.travis.yml

```diff
@@ -69,6 +69,14 @@ matrix:
       env: TOXENV=py34
     - python: "3.5"
       env: TOXENV=py35
+    - os: osx
+      osx_image: xcode7.3
+      sudo: required
+      language: generic
+      env: TOXENV=cover
+      before_install: "travis_retry brew install python augeas dialog"
```

Comment:
Coming from a Homebrew maintainer point of view: Please do run `brew update` when using Travis. Some Travis boxes are quite severely out of date & there's not much we can do to help if you hit issues running outdated versions of `brew`.